index.html

```diff
@@ -1,216 +1,189 @@
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&family=Pacifico&family=Rubik+Moonrocks&display=swap"
-      rel="stylesheet"
-    />
-    <link rel="stylesheet" href="styles.css" />
-    <!-- <link rel="stylesheet" href="animations.css" /> -->
-    <title>Portfolio Project</title>
-  </head>
 
-  <body>
-    <header class="top-bar">
-        <h4>Welcome</h4>
-      <nav class="hamburger">
-        <img
-          class="menu"
-          src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAARMAAAC3CAMAAAAGjUrGAAAAMFBMVEX///8AAADl5eX09PTV1dUICAiurq5TU1Orq6tJSUnc3Nzt7e1ERET6+vpRUVG9vb0gCS9GAAAA8ElEQVR4nO3dOQ4CQRAEQZblPv//W2yUFsLoNSKekGb1SLPbAQAAAAAAAAAAAADfbodh9+kC9ViGXaYL1Hm6yWm6QGlSmpQmpUlpUpqUJqVJaVKalCalSWlSmpQmpUlpUpqUJnWdbrLBjXqd9pwuAAAAAMCvzstx1Ba3x9f09rjBJjbq0qQ0KU1Kk9KkNClNSpPSpDQpTUqT0qQ0KU1Kk9KkNClvQet9mnW5TRcAAAAAgP/d98PW6QJljy27fWlSmpQmpUlpUpqUJqVJaVKalCalSWlSmpQmpUlpUpqUJuWPs3LLAAAAAAAAAAAAAGAbPhzZL1DD5UvKAAAAAElFTkSuQmCC"
-          alt="Menu Icon"
-        />
-      </nav>
-    </header>
-    <main>
-      <nav class="desktop-nav">
-        <nav class="item-nav">
-          <div id="nav-ptf" class="txt-nav">Portfolio</div>
-          <div id="nav-abt" class="txt-nav">About</div>
-          <div id="nav-cnt" class="txt-nav">Contact</div>
-        </nav>
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&family=Pacifico&family=Rubik+Moonrocks&display=swap"
+    rel="stylesheet" />
+  <link rel="stylesheet" href="styles.css" />
+  <!-- <link rel="stylesheet" href="animations.css" /> -->
+  <title>Portfolio Project</title>
+</head>
+
+<body>
+  <header class="top-bar">
+    <h4>Welcome</h4>
+    <nav class="hamburger">
+      <img class="menu"
+        src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAARMAAAC3CAMAAAAGjUrGAAAAMFBMVEX///8AAADl5eX09PTV1dUICAiurq5TU1Orq6tJSUnc3Nzt7e1ERET6+vpRUVG9vb0gCS9GAAAA8ElEQVR4nO3dOQ4CQRAEQZblPv//W2yUFsLoNSKekGb1SLPbAQAAAAAAAAAAAADfbodh9+kC9ViGXaYL1Hm6yWm6QGlSmpQmpUlpUpqUJqVJaVKalCalSWlSmpQmpUlpUpqUJnWdbrLBjXqd9pwuAAAAAMCvzstx1Ba3x9f09rjBJjbq0qQ0KU1Kk9KkNClNSpPSpDQpTUqT0qQ0KU1Kk9KkNClvQet9mnW5TRcAAAAAgP/d98PW6QJljy27fWlSmpQmpUlpUpqUJqVJaVKalCalSWlSmpQmpUlpUpqUJuWPs3LLAAAAAAAAAAAAAGAbPhzZL1DD5UvKAAAAAElFTkSuQmCC"
+        alt="Menu Icon" />
+    </nav>
+  </header>
+  <main>
+    <section class="firth">
+    <nav class="desktop-nav">
+      <nav class="item-nav">
+        <div id="nav-ptf" class="txt-nav">Portfolio</div>
+        <div id="nav-abt" class="txt-nav">About</div>
+        <div id="nav-cnt" class="txt-nav">Contact</div>
```

Comment:
- [OPTIONAL] Kindly wrap each element inside an `a` tag and please make sure that every link redirects the user to the different parts of your portfolio. :+1: :smiley: :smiley: 

![image](https://user-images.githubusercontent.com/98363075/193183057-ec594d44-f4d3-429d-a38f-af8fd1be5b9c.png)